source/_integrations/yweather.markdown

```diff
@@ -11,7 +11,7 @@ ha_domain: yweather
 
 <div class='note warning'>
 
-The Yahoo Weather API is being [retired](https://developer.yahoo.com/weather/?guccounter=1). A replacement is the [`darksky` sensor](/integrations/darksky).
+The Yahoo Weather API is being [retired](https://developer.yahoo.com/weather/?guccounter=1). A replacement is the [`NWS` integration](/integrations/nws).
```

Comment:
I suggest to choose a replacement which is not limited to the US.

Reply:
Yes, that is a very good point.  I believe OpenWeatherMap would be a better suggestion.